tools/api-markdown-documenter/README.md

```diff
@@ -16,7 +16,7 @@ One may be added in the future, but for now this library is intended to be consu
 The following terms are leveraged heavily in this package's APIs and documentation.
 
 -   **API Model**: Refers to a complete API suite, comprised of one or more packages.
-    This often corresponds to all of the packages in the mono-repo, or a series of packages that are published together.
+    This often corresponds to all of the packages in the mono-repo, or a series of packages whose API docs are published together.
```

Comment:
The previous text here was incorrect. It doesn't matter if the packages themselves are published together - it only matters if their _docs_ are published together.